pkg/scheduler/framework/runtime/framework.go

```diff
@@ -247,6 +254,8 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 		return f, nil
 	}
 
+	profile := config.KubeSchedulerProfile{SchedulerName: f.ProfileName(), Plugins: plugins, PluginConfig: make([]config.PluginConfig, 0)}
```

Comment:
If the goal is to print the final config, we missed a little thing, at line 296, the weight changed to 1 if it is 0. Not sure it is worth it or not. Actually, I also point it out at https://github.com/kubernetes/kubernetes/pull/96438/commits/d6ef6d6e433f965dd0b25f226c3fe5ef411ee553#r521218103, at least it is fine for the test.